src/main/kotlin/by/architecture_map/belarus/service/impl/ArchitectServiceImpl.kt

```diff
@@ -0,0 +1,25 @@
+package by.architecture_map.belarus.service.impl
+
+import by.architecture_map.belarus.entity.Architect
+import by.architecture_map.belarus.repository.ArchitectRepository
+import by.architecture_map.belarus.service.ArchitectService
+import org.springframework.data.repository.findByIdOrNull
+import org.springframework.stereotype.Service
+
+@Service
+class ArchitectServiceImpl(
+        private val architectRepository: ArchitectRepository
+) : ArchitectService {
+
+    override fun create(architect: Architect): Architect {
+        return architectRepository.save(architect)
```

Comment:
in most cases we do not use `return` statement in Kotlin, like this:
```
override fun create(architect: Architect) =
       architectRepository.save(architect)
```

you can also skip the return type